src/challonge.service.js

```diff
@@ -1,16 +1,8 @@
 const R = require('ramda');
-const axios = require('axios');
-
-function challongeServiceFactory({ organization, apiKey }) {
-    const challongeApi = axios.create({
-        baseURL: 'https://api.challonge.com/v1/',
-        params: {
-            api_key: apiKey,
-        },
-    });
 
+function challongeServiceFactory({ api, organization }) {
```

Comment:
Why not change this module like `user.repository.js`, injecting api / org into each repo method separately?

Reply:
That would get mucky with the deps between service methods, though...

Reply:
Jepp, I tried to do it like in the user repo, but realized that the functions were not independent of each other